package/network/services/ppp/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=ppp
-PKG_RELEASE:=3
+PKG_RELEASE:=4
```

Comment:
Can you change this to autorelease api (if you don't know what i'm talking about feel free to ask) in another commit? (ideally before this?)

Reply:
(example https://github.com/openwrt/openwrt/blob/master/package/network/config/qos-scripts/Makefile#L12)